hw5_calculator_oop/HW5_calculator.mjs

```diff
@@ -0,0 +1,28 @@
+import { Calculator } from "./HW5_calculator_class.mjs";
+
+const calculator = new Calculator();
+console.log(`Add example: 14 + 41 = ${calculator.add(14, 41)}`);
+console.log(`Subtract example: 174 - 32 = ${calculator.subtract(174, 32)}`);
+console.log(`Multiply example: 47 * 50 = ${calculator.multiply(47, 50)}`);
+console.log(`Divide example: 125 / 7 = ${calculator.divide(125, 7)}`);
+
+//Also I have this idea, but I am not sure, if it is also ok for your assignment of homework
+
+/*import { Calculator } from "./HW5_calculator_class.mjs";
+
+const calculator = new Calculator();
+
+const number1 = 14;
```

Comment:
Hello, this is pretty much ok solution but it would be better to use more specific variables (addNumber1, multiplyNumber1 etc.)